app/router/schema.js

```diff
@@ -53,6 +65,13 @@ exports.push = async (req, res) => {
 		})
 	);
 
+	if (config.asyncSchemaUpdates) {
+		await kafkaProducer.send({
+			topic: 'test-topic',
+			messages: [{ value: 'Schema Updated!' }],
+		});
+	}
+
```

Comment:
`composeLatest` is intended for simplistic reading of current schema (mapped by service) by the gateway.. so its not the right location to produce an event

Federated services need to call `/schema/push`, so I would imagine that such event would need to be somewhere there